receipt/SaleReceipt_fr_CA-QC.tpl

```diff
@@ -1143,7 +1156,15 @@ table.payments td.label {
 				<td class="amount">{{Payment.amount|money}}</td>
 			{% elseif Payment.ccChargeID == 0 %}
 				<td class="label" width="100%">
-					{{ Payment.PaymentType.name }}
+                    {% if Payment.PaymentType.name == 'Credit Card' %}
+                        Carte de crédit
+                    {% elseif Payment.PaymentType.name == 'Debit Card' %}
+                        Carte de débit
+                    {% elseif Payment.PaymentType.name == 'Check' %}
+                        Chèque
+                    {% else %}
+                        {{Payment.PaymentType.name}}
+                    {% endif %}
```

Comment:
Same, again sorry for asking, if you can run prettier on those files in another PR I'd gladly approve it